lib/models/user.js

```diff
@@ -102,29 +235,25 @@ User.statics.create = function(email, passwd, callback) {
   // Check to make sure the password is already SHA-256 (or at least is the
   // correct number of bits).
   if (Buffer(passwd, 'hex').length * 8 !== 256) {
-    return callback(
-      new errors.BadRequestError('Password must be hex encoded SHA-256 hash')
-    );
+    return callback(new Error('Password must be hex encoded SHA-256 hash'));
```

Comment:
These bubble up to router and need to include the proper status code? Why removed?